evalai/utils/common.py

```diff
@@ -31,6 +34,36 @@ def convert(self, value, param, ctx):
             )
 
 
+def upload_with_presigned_url(file, presigned_url):
+    """Function to upload a file, given the target presigned s3 url.
```

Comment:
Fix the docstring

Reply:
Fix how? Move it to the next line? I'm following this, as Rishabh suggested this in EvalAI repository for my other PR: https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Reply:
See this for an example https://github.com/Cloud-CV/EvalAI/blob/master/apps/base/utils.py#L152